### ISSUE TITLE
[BUG] Improve error message

### ISSUE THREAD

**Issue by MarleenGilsing** [hedyorg/hedy#4672]
I found this error message, that I think can be better. 

![image](https://github.com/hedyorg/hedy/assets/80678586/ec53716a-444c-447e-b4bf-d4df7d4a06a3)

**Comment by MarleenGilsing** [comment]
There's more

![image](https://github.com/hedyorg/hedy/assets/80678586/f8e3f4d6-cf0c-4972-bbd4-ab5703922079)


And more...

![image](https://github.com/hedyorg/hedy/assets/80678586/df631325-2014-46cd-801e-9450d6becc59)
"You can't use a number, try changing it to a number"

**Comment by MarleenGilsing** [comment]
And something very weird! It works with the variable 'tim' but not with 'time'???

This works:
![image](https://github.com/hedyorg/hedy/assets/80678586/c6d07fb5-dd62-43a8-87d9-9bede332dddc)

This doesn't: 
![image](https://github.com/hedyorg/hedy/assets/80678586/4d2cdb77-c883-4d05-a461-e0dc6e89e63f)

**Comment by Felienne** [comment]
> And something very weird! It works with the variable 'tim' but not with 'time'???
> 
> This works: ![image](https://user-images.githubusercontent.com/80678586/278285100-c6d07fb5-dd62-43a8-87d9-9bede332dddc.png)
> 
> This doesn't: ![image](https://user-images.githubusercontent.com/80678586/278285287-4d2cdb77-c883-4d05-a461-e0dc6e89e63f.png)

Good catch and thanks for diving in deeper, this is most likely caused by `time` being a "special word" in Python! I will pick this up!
